tailcall-http-cache/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+mod http_cache_manager;
+
+pub use http_cache_manager::HttpCacheManager;
```

Comment:
```suggestion
pub use cache::HttpCacheManager;
```

Can keep the name simple

Reply:
done, renamed module to cache.